interface.html

```diff
@@ -130,3 +146,102 @@ <h4 class="panel-title" data-toggle="collapse" data-parent="#accordion" data-tar
     </div>
   </div>
 </template>
+
+<template id="style-default">
```

Comment:
these templates could go into the handlebars templates: https://github.com/Fliplet/fliplet-cli#using-handlebars-in-your-client

Example:
https://github.com/Fliplet/fliplet-widget-chat/tree/master/templates
https://github.com/Fliplet/fliplet-widget-chat/blob/master/js/build.js#L132